### ISSUE TITLE
Redesign

### ISSUE THREAD

**Issue by jhnns** [webpack/docs#69]
#64 @gregvenech

**Comment by gregvenech** [comment]
Here's another shot of the documentation (updated slightly):

![documentation](https://cloud.githubusercontent.com/assets/8988697/16030648/531310ae-31c1-11e6-83e8-0447382f8cac.png)

And one of the splash page I've been playing around with:

![splash](https://cloud.githubusercontent.com/assets/8988697/16030695/da8684f8-31c1-11e6-9886-4c00c6a554d5.png)

I'm very happy with the way the documentation is looking although I am still planning to tweak the font size a bit as recommended by @jhnns and @bebraw in #64. I'm also in the process of putting together a codepen to play around with logotypes and the cubes, I'll share this soon...

The splash page is on it's way there but there are some things I'm still working on:

- Too much blank space in the header section. Those animated-vectory backgrounds, like the one in the top of [this grunt redesign](https://dribbble.com/shots/1471385-Grunt/attachments/219142), could work but it would be cool if we could think of something that has more to do with Webpack than some random landscape.
- The modules => output graphic. Need to actually draw new arrows and maybe add a few more modules?
- Update font size here as well to correspond with the documentation, possibly using the [modular scale](http://zellwk.com/blog/modular-scale/) (thanks @jhnns).

I pulled the "Webpack, the flexible..." section from [here](http://webpack.github.io/docs/) as I think it fits better in the Splash page than in the documentation (might be cool if we randomly cycle the adjectives through as well).

Anyway just some thoughts... suggestions welcome

**Comment by jhnns** [comment]
Just a short brain dump: 😉 

- I really like the dependency graph. I think, we could use that one as illustration. We can also play with the graph/mesh look to create a nice background pattern. I think, webpack can be illustrated nicely as nodes and edges because that's what the dependency graph is about.
- I've thought about the logo for some time but personally I would not change it completely anymore. I think, people are already used to it and I really like the shape. I'd rather try to polish it: add some contrast, play with colors, etc.
- The current logo could be used to create a nice pattern because you can repeat the cube infinitely. Maybe that's a nice background instead of the random landscape (although random landscapes seem to be really in vogue :wink:)
- Personally I don't really like the "Webpack, the flexible..." section. Donnu why 😁. I would drop that for the new website.

**Comment by gregvenech** [comment]
> I really like the dependency graph. I think, we could use that one as illustration. We can also play with the graph/mesh look to create a nice background pattern.

Great idea, I didn't even think of using that as a background. Makes sense though as it would be more relevant than some landscape (even if they are in vogue).

> Personally I don't really like the "Webpack, the flexible..." section. Donnu why 😁. I would drop that for the new website.

Ahhh man, I was feeling like maybe something's missing there but I thought it would be nice to have a kind of catch-phrase section. Maybe there's something I could tweak to sell you on it... it's definitely not make or break though 😉

**Comment by gregvenech** [comment]
Here's [that pen][1] I was talking about... I put it together based on the existing spin behavior and colors both from the current site and [this site][2]. In the end though they should match up with the color scheme from the general redesign or vice versa if we choose new ones. Obviously feel free to fork it and share back... @jhnns I know you might have some ideas.

I'm reading through the [modular scale][3] article and making some other tweaks to the mockups. I was thinking of putting together a simple repo to start playing around with the new layout and design elements. In terms of rebuilding here are my thoughts:

> For the docs it's probably a good idea to start from scratch and just ignore the old content.

- I think starting from scratch, as mentioned by @sokra in #64, is probably easier than trying to refactor this repository. I know there's still more discussion to be had in terms of restructuring but I think getting the general site structure/layout down would help kick things off.
- It would be cool if we could use Webpack as the only build tool. I know this repository uses grunt to do a static site build but I feel we can simplify that by processing wiki content on the fly using a library like [marked][4] which would also mean it's always up to date. 
- I've been going back and forth on whether to use React as the base framework. I did use it in the [pen][1] but I feel like too many of the Webpack tutorials I've seen couple Webpack with React to the point where it's hard to see how great Webpack is even if you aren't using that particular library. On the other hand I feel like it's stupid not to use whatever makes the most sense.

If any of this is going too far, please let me know. I'm willing to contribute however I can. It sounds like the content will be coming from a variety of sources which I know might complicate things: the wiki, @bebraw's book, and possibly some generated by JSDoc as mentioned by @also .

[1]: http://codepen.io/gvenech/full/2323a1368d0e1e18c5ddaf899f4017ef/
[2]: https://flatuicolors.com/
[3]: http://zellwk.com/blog/modular-scale/
[4]: https://www.npmjs.com/package/marked

**Comment by gregvenech** [comment]
Updated the link to [the logo pen](http://codepen.io/gvenech/pen/OXNEbZ/), sorry didn't realize it was private.

**Comment by jhnns** [comment]
@gregvenech thx for your contributions at webpack.io. The logo looks great, I like how the cube rotates different every time.

We will talk about the foundations of webpack.io in one of our upcoming meetings.

**Comment by gregvenech** [comment]
@jhnns no problem, yeah I'm pretty happy with the way it came out but I still think the colors and maybe the color scheme in general might need some tweaking.

My plan was to start on the splash page next since it seems like there's still some discussion to be had on the rest of the content... I can hold off on it though if you'd like me to wait until after the meeting?

**Comment by bebraw** [comment]
The meeting is literally in fifteen minutes so it shouldn't take a lot of waiting perhaps. 👍

**Comment by Shinobi881** [comment]
Tonight, I gave an intro to Webpack talk at the Real World React meetup in SF. There had to be at least a couple hundred people there. The most awesome part was how so many people (both experienced and new) came up to me afterward to say "I didn't realize it was so easy to get started". Many were saying that their companies were migrating to Webpack. 

Personally, I got started with the "Getting Started" and thought it was a pretty good starting point. I just wanted to thank you all for the awesome work you have done and once I land a job and get my bills paid. I look forward to contributing.

Either way, I'm looking to get started on some in-person workshops, here in the Bay Area. #MultiprongedApproach.
